components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -433,7 +433,22 @@ class PageWallet extends React.Component<Props, State> {
     return getLocale('greetingsVerified', { name: externalWallet.userName })
   }
 
+  disconnectWalletAlert = (): AlertWallet | null => {
+    const { ui, externalWallet } = this.props.rewardsData
+    const walletType = externalWallet ? externalWallet.type : ""
+    const showDisconnectWalletError = ui && ui.disconnectWalletError
+    if (showDisconnectWalletError) {
+      return {
+        node: <><b>{getLocale('uhOh')}</b><br />{getLocale('disconnectWalletFailed', { wallet: walletType })}</>,
```

Comment:
When you change to use message placeholders instead of `[[x]]`, we'll need to do something like:

```js
getLocale('disconnectWalletFailed').replace('$1', walletType)
```